vivo-project/VIVO
# Correct 10 broken prefixes

This pull request addresses issue [VIVO-1548](https://jira.duraspace.org/browse/VIVO-1548)

# What does this pull request do?
Corrects 10 broken prefixes in vitroAssertions.n3 introduced in VIVO 1.10

The broken prefixes cause annotations on the effected ontologies to be unfindable from the application.  See file change for affected ontologies.

# What's new?
Prefixes are corrected.  Annotations for the effected ontologies can now be found.

# How should this be tested?
1. Build 1.11 from scratch.  Run two queries shown below.  You will see that skos:Concept, skos being one of the affected ontologies, now has ontology assertions looking good, but annotation assertions made with a wrong URI.
2. Build 1.11 with this pull request.  Run two queires as shown below.  You will see that skos:Concept, skos being one of the affected ontologies, now has ontology assertions and annotations looking good. No annotation assertions made with a wrong URI.

## Before

```
    SELECT ?s ?p ?o
    WHERE {
      <http://www.w3.org/2004/02/skos/core#Concept> ?p ?o .
    }
```
produces
```
---------------------------------------------------------------------------------------------------------
| s | p                                                 | o                                             |
=========================================================================================================
|   | <http://www.w3.org/2002/07/owl#equivalentClass>   | <http://www.w3.org/2004/02/skos/core#Concept> |
|   | <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> | <http://www.w3.org/2002/07/owl#Class>         |
|   | <http://www.w3.org/2000/01/rdf-schema#subClassOf> | <http://www.w3.org/2002/07/owl#Thing>         |
|   | <http://www.w3.org/2000/01/rdf-schema#label>      | "Concept"@en                                  |
---------------------------------------------------------------------------------------------------------
```

while

```
    SELECT ?p ?o
    WHERE {
      <http://www.w3.org/2004/02/skos/coreConcept> ?p ?o .
    }
```
produces
```
-------------------------------------------------------------------------------------------------------------------------------------------------------
| p                                                                                       | o                                                         |
=======================================================================================================================================================
| <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#hiddenFromDisplayBelowRoleLevelAnnot>    | <http://vitro.mannlib.cornell.edu/ns/vitro/role#public>   |
| <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#prohibitedFromUpdateBelowRoleLevelAnnot> | <http://vitro.mannlib.cornell.edu/ns/vitro/role#public>   |
| <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#inClassGroup>                            | <http://vivoweb.org/ontology#vitroClassGrouppublications> |
| <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#customDisplayViewAnnot>                  | "individual--skos-concept.ftl"                            |
| <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#displayLimitAnnot>                       | "-1"^^<http://www.w3.org/2001/XMLSchema#int>              |
| <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#hiddenFromPublishBelowRoleLevelAnnot>    | <http://vitro.mannlib.cornell.edu/ns/vitro/role#public>   |
| <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#displayRankAnnot>                        | "35"^^<http://www.w3.org/2001/XMLSchema#int>              |
-------------------------------------------------------------------------------------------------------------------------------------------------------
```

## After

```
SELECT ?s ?p ?o
WHERE {
  <http://www.w3.org/2004/02/skos/core#Concept> ?p ?o .
}
```

produces

```
-----------------------------------------------------------------------------------------------------------------------------------------------------------
| s | p                                                                                       | o                                                         |
===========================================================================================================================================================
|   | <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#hiddenFromDisplayBelowRoleLevelAnnot>    | <http://vitro.mannlib.cornell.edu/ns/vitro/role#public>   |
|   | <http://www.w3.org/2002/07/owl#equivalentClass>                                         | <http://www.w3.org/2004/02/skos/core#Concept>             |
|   | <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#prohibitedFromUpdateBelowRoleLevelAnnot> | <http://vitro.mannlib.cornell.edu/ns/vitro/role#public>   |
|   | <http://www.w3.org/1999/02/22-rdf-syntax-ns#type>                                       | <http://www.w3.org/2002/07/owl#Class>                     |
|   | <http://www.w3.org/2000/01/rdf-schema#subClassOf>                                       | <http://www.w3.org/2002/07/owl#Thing>                     |
|   | <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#inClassGroup>                            | <http://vivoweb.org/ontology#vitroClassGrouppublications> |
|   | <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#customDisplayViewAnnot>                  | "individual--skos-concept.ftl"                            |
|   | <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#displayLimitAnnot>                       | "-1"^^<http://www.w3.org/2001/XMLSchema#int>              |
|   | <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#hiddenFromPublishBelowRoleLevelAnnot>    | <http://vitro.mannlib.cornell.edu/ns/vitro/role#public>   |
|   | <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#displayRankAnnot>                        | "35"^^<http://www.w3.org/2001/XMLSchema#int>              |
|   | <http://www.w3.org/2000/01/rdf-schema#label>                                            | "Concept"@en                                              |
-----------------------------------------------------------------------------------------------------------------------------------------------------------
```

while


```
SELECT ?p ?o
WHERE {
  <http://www.w3.org/2004/02/skos/coreConcept> ?p ?o .
}
```

produces an empty result

# Additional Notes:
Important bug

# Interested parties
@vivo-project/vivo-committers
